recipe/meta.yaml

```
@@ -11,7 +11,7 @@ package:
 
 source:
   url: https://archive.apache.org/dist/arrow/arrow-{{ version }}/{{ filename }}
-  sha256: 9d280d8042e7cf526f8c28d170d93bfab65e50f94569f6a790982a878d8d898d
+  sha256: f89b93f39954740f7184735ff1e1d3b5be2640396febc872c4955274a011f56b
 
 build:
   number: 1
```

```suggestion
  number: 0
```
Because we update to a new version

---

done